### DISCUSSION TITLE
How to replace /local/path/data ?

### DISCUSSION THREAD

**Discussion by oitkenmen** [Budibase/budibase#12259] [Help] [answered]
Hi, I'm trying to install budibase using Docker on Windows 10. Accroding to [this page](https://docs.budibase.com/docs/docker) I need to replace /local/path/data with some absolute path. What should I write here?

**Comment by ConorWebb96** [comment]
Hey @nicktoa,

You can use any path I believe as long as its a correct and exists on your machine. For example mines is `Desktop/budi/single-image`. Just make sure the path used exist and is correct.

**Reply by oitkenmen** [reply]
Thanks!

**Reply by oitkenmen** [reply]
Do I need to write
  `-v Desktop/budi/single-image:/data \`
instead
  `-v /local/path/data:/data \`
?

**Reply by ConorWebb96** [reply]
As long as thats a valid path it should work. This isn't really a Budibase question, its more a docker [question](https://docs.docker.com/storage/bind-mounts/) you can read up more on this with the link provided.

I hope this helps.

**Comment by oitkenmen** [comment]
I've installed Docker Desktop and pulled the Budibase image. Now I'm trying to run
```
docker run -d -t \
  --name=budibase \
  -p 10000:80 \
  -v /local/path/data:/data \
  --restart unless-stopped \
  budibase/budibase:latest
```
as it was written [here](https://docs.budibase.com/docs/docker).
But then I receive errors:

```
At line:2 char:5
+   --name=budibase \
+     ~
Missing expression after unary operator '--'.
At line:2 char:5
+   --name=budibase \
+     ~~~~~~~~~~~~~
Unexpected token 'name=budibase' in expression or statement.
At line:3 char:5
+   --hostname=budibase \
+     ~
Missing expression after unary operator '--'.
At line:3 char:5
+   --hostname=budibase \
+     ~~~~~~~~~~~~~~~~~
Unexpected token 'hostname=budibase' in expression or statement.
At line:7 char:5
+   --restart unless-stopped \
+     ~
Missing expression after unary operator '--'.
At line:7 char:5
+   --restart unless-stopped \
+     ~~~~~~~
Unexpected token 'restart' in expression or statement.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : MissingExpressionAfterOperator
```

What I'm doing wrong here? Can somebody please provide a clear explanation how to run this.

**Reply by oitkenmen** [reply, marked as answer]
Okay, now I understand. There's no need to write any commands. You just need to click the "Run" button. Then, Docker will provide the options to fill. All ports are "0" except for port 80, which is "10000". The name is "budibase". That's it. Then it runs.
